docs/test/deploy-from-settings.md

```diff
@@ -0,0 +1,47 @@
+# Deploy from settings
+
+## Prerequisites
+Open deploy dialog (+) and select "Specify app details below"
+
+## Expected result at execution
+* In positive case the "Deploy" button must lead to the creation of a running application
+* Validation errors affecting particular fields must be displayed beneath these fields. In case the deployment  fails due to an exception, then a pop dialog is shown
+
+## Specific factors
+
+|Factor                      |1                 |2                                 |3               |4               |5                        |6                    |7               |Comment|
+|----------------------------|------------------|----------------------------------|----------------|----------------|-------------------------|---------------------|----------------|-------|
+|App name                    |Empty             |Correct value                     |Max-length      |Validation error|Same name app exists     |                     |                |       |
+|Container image             |Empty             |Correct value                     |Max-length      |With version    |With hostname of registry|With port of registry|Validation error|       |
+|Number of pods              |0                 |Positive integer                  |Negative integer|Not integer     |Not a number             |                     |                |       |
+|Port                        |0                 |Positive integer                  |Negative integer|Not integer     |Not a number             |                     |                |       |
+|Target Port                 |0                 |Positive integer                  |Negative integer|Not integer     |Not a number             |                     |                |       |
+|Protocol                    |TCP               |UDP                               |                |                |                         |                     |                |       |
+|Expose service externally   |Checked           |Not checked                       |                |                |                         |                     |                |       |
+|Description                 |Empty             |Correct value                     |Max-length      |Validation error|                         |                     |                |       |
```

Comment:
hmm..how is a description mapped to k8s resources? I could not retrieve it with kubectl. Can you investigate? Could be interesting to create factors